### PR TITLE
Index hackathon

### PR DESCRIPTION
**Pull request by bellemae**
Documentation changes stemming from ODC conf discussions

### CODE REVIEW

File: docs/ops/prepare_scripts.rst [hunk 1]
```diff
@@ -16,7 +16,8 @@ tools are provided in  :file:`utils/` in the source repository.
 
 The two examples below shows USGS landsat data for ingestion into the Data cube.
 
-#. A specific example for USGS collection 1 MTL format is :download:`USGS Landsat Prepare Script
+
+1. A specific example for USGS collection 1 MTL format is :download:`USGS Landsat Prepare Script
```
Comment (andrewdhicks) [inline docs/ops/prepare_scripts.rst]: Should this be moved under the heading below?
If the numbering of the two examples is required, it should probably be the headings that are numbered.

Comment (andrewdhicks) [reply docs/ops/prepare_scripts.rst]: Also, an 's' is missing from a couple of lines down: "any of the link below:"

File: docs/ops/prepare_scripts.rst [hunk 2]
```diff
@@ -66,7 +67,7 @@ For Landsat collection 1 level 1 product:
 
 
 
-#. An another example for USGS landsat surface reflectance :download:`USGS Landsat LEDAPS
+2. An another example for USGS landsat surface reflectance :download:`USGS Landsat LEDAPS
```
Comment (andrewdhicks) [inline docs/ops/prepare_scripts.rst]: See above

File: docs/ops/indexing.rst [hunk 3]
```diff
@@ -14,38 +14,27 @@ existence of and detailed metadata about the data into the **index**. None of
...
-__ https://lpdaac.usgs.gov/dataset_discovery/modis/modis_products_table/mcd43a1
-__ https://lpdaac.usgs.gov/dataset_discovery/modis/modis_products_table/mcd43a2
-__ https://lpdaac.usgs.gov/dataset_discovery/modis/modis_products_table/mcd43a3
-__ https://lpdaac.usgs.gov/dataset_discovery/modis/modis_products_table/mcd43a4
+Steps to Adding Data
+====================
 
-Once you have downloaded some data, it will need :ref:`metadata preparation
-<prepare-scripts>` before use in the Data Cube.
+ * Create a new product
```
Comment (andrewdhicks) [inline docs/ops/indexing.rst:30]: This is a great improvement to the docs!

**Comment by woodcockr** [comment]
I'd be happy to review this pull request if you'd like to share the load?
I was involved with it at the start so familiar with the intent.

**Comment by harshurampur** [comment]
@woodcockr No problem!

**Review by andrewdhicks — CHANGES_REQUESTED** [review]
The NDVI section of the datacube-indexing notebook could be removed.
We eventually want to move the crontrib notebooks out of the core repo into a gallery repo, once the CEOS UI issues are sorted and the DEA gallery trial has finished.

File: utils/ls_public_bucket.py [hunk 4]
```diff
@@ -0,0 +1,217 @@
+# coding: utf-8
```
Comment (omad) [inline utils/ls_public_bucket.py:1]: This looks like a handy utility, but it'd be great to have a docstring here explaining what it does, and giving an example of running it.

Could be similar, but not exactly what's talked about in the notebook.

**Comment by harshurampur** [comment]
@andrewdhicks Thanks for having a detailed look and I have updated all the changes.

**Review by omad — CHANGES_REQUESTED** [review]
Looks good, but I'd like some more information included inside s3 indexing utility about what it is and how to use it.

The notebook covers is super useful, but I'm not so keen on the 2.2Mb it adds to the repository, almost entirely from the Images at the bottom.

Did we ever decide where to put weighty notebooks?
I think for indexing data they're pretty, but don't add much to the explanation.

File: utils/ls_public_bucket.py (hunk 4 above)
Comment (harshurampur) [reply utils/ls_public_bucket.py:1]: Yes, that's right. I will add all the information on how to use it ..

File: docs/ops/indexing.rst (hunk 3 above)
Comment (woodcockr) [reply docs/ops/indexing.rst:30]: Agreed, much improved.

File: docs/ops/indexing.rst [hunk 5]
```diff
@@ -68,17 +57,79 @@ what type of **units** the measurement is in, a **nodata** value, and even a way
 of specifying **bit level descriptions** or the **spectral response** in the
 case of reflectance data.
 
-A set of example Product definitions are supplied in
-`docs/config_samples/dataset_types` to cover some common Geoscience Australia
-and other Earth Observation Data.
+A product definition example:
```
Comment (woodcockr) [inline docs/ops/indexing.rst:61]: I did wonder if an even simpler version could be done, but then thought this one has the merit of being both real and very simple even if it is a bit long if you only want one band.

File: utils/ls_public_bucket.py (hunk 4 above)
Comment (woodcockr) [reply utils/ls_public_bucket.py:1]: @omad said it all!

**Review by woodcockr — DISMISSED** [review]
Great improvements. Reads well to me.
Answers the key question we get asked:
1. Where do I get data?
2. How to I configure a simple index?
3. What is the difference between index and ingest?

**Review by omad — DISMISSED** [review]
👍

**Review by andrewdhicks — APPROVED** [review]
:+1:

**Pull request merged**